tests/integration/test_store.py

```diff
@@ -1340,8 +1343,8 @@ def test_span_ingestion(
                         "traceId": "89143b0763095bd9c9955e8175d1fb23",
                         "spanId": "e342abb1214ca181",
                         "name": "my 1st OTel span",
-                        "startTimeUnixNano": 1697620454980000000,
-                        "endTimeUnixNano": 1697620454980078800,
+                        "startTimeUnixNano": int(start.timestamp() * 1_000_000_000),
+                        "endTimeUnixNano": int(end.timestamp() * 1_000_000_000),
```

Comment:
Our clock drift processor fixes very old timestamps, so we have to use dynamic timestamps here.